sources/Core/Numerics/Vector3.cs

```diff
@@ -70,6 +77,73 @@ public Vector3(float x, float y, float z)
+        /// <returns>The result of subtracting <paramref name="right"/> from <paramref name="left"/>.</returns>
+        public static Vector3 operator -(Vector3 left, Vector3 right) => new Vector3(left.X - right.X, left.Y - right.Y, left.Z - right.Z);
+
+        /// <summary>Multiplies two specified <see cref="Vector3"/> values.</summary>
+        /// <param name="left">The first value to multiply.</param>
+        /// <param name="right">The second value to multiply.</param>
+        /// <returns>The result of multiplying <paramref name="left"/> by <paramref name="right"/>.</returns>
+        public static Vector3 operator *(Vector3 left, Vector3 right) => new Vector3(left.X * right.X, left.Y * right.Y, left.Z * right.Z);
+
+        /// <summary>Divides two specified <see cref="Vector3"/> values.</summary>
+        /// <param name="left">The dividend.</param>
+        /// <param name="right">The divisor.</param>
+        /// <returns>The result of dividing <paramref name="left"/> by <paramref name="right"/>.</returns>
+        public static Vector3 operator /(Vector3 left, Vector3 right) => new Vector3(left.X / right.X, left.Y / right.Y, left.Z / right.Z);
+
+        /// <summary>Multiplies each component of a <see cref="Vector3"/> value by a given <see cref="float"/> value.</summary>
+        /// <param name="left">The vector to multiply.</param>
+        /// <param name="right">The value to multiply each component by.</param>
+        /// <returns>The result of multiplying each component of <paramref name="left"/> by <paramref name="right"/>.</returns>
+        public static Vector3 operator *(Vector3 left, float right) => new Vector3(left.X * right, left.Y * right, left.Z * right);
+
+        /// <summary>Divides each component of a <see cref="Vector3"/> value by a given <see cref="float"/> value.</summary>
+        /// <param name="left">The dividend.</param>
+        /// <param name="right">The divisor to divide each component by.</param>
+        /// <returns>The result of multiplying each component of <paramref name="left"/> by <paramref name="right"/>.</returns>
+        public static Vector3 operator /(Vector3 left, float right) => new Vector3(left.X / right, left.Y / right, left.Z / right);
+
+        /// <summary>Computes the cross product of two <see cref="Vector3"/> values.</summary>
+        /// <remarks>This method assumes both vectors <paramref name="left"/> and <paramref name="right"/> start at the origin.</remarks>
+        /// <param name="left">The first value to cross.</param>
+        /// <param name="right">The second value to cross.</param>
+        /// <returns>The cross product of <paramref name="left"/> and <paramref name="right"/></returns>
+        public static Vector3 Cross(Vector3 left, Vector3 right) => new Vector3(
+            (left.Y * right.Z) - (left.Z * right.Y),
+            (left.Z * right.X) - (left.X * right.Z),
+            (left.X * right.Y) - (left.Y * right.X));
```

Comment:
I personally prefer the following and find it slightly more parseable:
```suggestion
            (left.X * right.Y) - (left.Y * right.X)
        );
```